CanHazFunny/CanHazFunny/IJokesOut.cs

```diff
@@ -0,0 +1,5 @@
+namespace CanHazFunny;
+public interface IJokesOut
+{
+    public void PrintJoke(string? joke);
```

Comment:
```suggestion
    public void PrintJoke(string joke);
```
This probably doesn't need to be nullable.